templates/sample-app/res/src/node/lib/handler/createData/sampleData/Contact.json

```diff
@@ -0,0 +1,4119 @@
+{
+	"records": [{
+		"FirstName": "Mission High School",
```

Comment:
A Contact called Mission High School Mission High School? Mean parents! :P

By the way, First Name is optional, Last Name is required, you can just leave the First Names blank and remove some duplication.